### PR TITLE
fix: Fixed the issue in social-media template related to formation of circular json.

### PR DESCRIPTION
**Pull request by Hamza-777**
Description:
- On liking and bookmarking the same post circular json formation error was being thrown.
- Fixed the issue by adding a user to likedBy and dislikedBy array with empty bookmarks array.
- This solves the issue and makes the template bug free.
- Changed comment delete handler listed as post in server.js file to delete

### CODE REVIEW

**Comment by sohamsshah** [comment]
Hi @Hamza-777 , thanks for submitting the PR, appreciate it! We will be reviewing and merging it shortly!

File: apps/cra-template-mockbee-social-media/template/src/server.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ export function makeServer({ environment = "development" } = {}) {
         "/comments/edit/:postId/:commentId",
         editPostCommentHandler.bind(this)
       );
-      this.post(
+      this.delete(
```
Comment (NadaaFarook) [inline apps/cra-template-mockbee-social-media/template/src/server.js:90]: This change is good. But the documentation will be needed to be updated too. Else the PR looks good to go

Comment (Hamza-777) [reply apps/cra-template-mockbee-social-media/template/src/server.js:90]: Ok will do that

Comment (Hamza-777) [reply apps/cra-template-mockbee-social-media/template/src/server.js:90]: Is that something I can do or yoj will?

Comment (sohamsshah) [reply apps/cra-template-mockbee-social-media/template/src/server.js:90]: Hey @Hamza-777 , you can make that change!

Comment (Hamza-777) [reply apps/cra-template-mockbee-social-media/template/src/server.js:90]: @NadaaFarook The docs for this change are correct already.

Comment (Hamza-777) [reply apps/cra-template-mockbee-social-media/template/src/server.js:90]: On it.

Comment (Hamza-777) [reply apps/cra-template-mockbee-social-media/template/src/server.js:90]: @sohamsshah Made the required changes in the docs.

**Comment by sohamsshah** [comment]
Thanks for the PR!

**Comment by Hamza-777** [comment]
> Thanks for the PR!

Pleasure's all mine.... 👍

**Pull request merged**